doc/databinding/index.md

```diff
@@ -0,0 +1,53 @@
+---
+---
+# Data Binding
+
+## Introduction
+
+Tabris provides a [JSX](../JSX.md)/[decorators](http://www.typescriptlang.org/docs/handbook/decorators.html)-based declarative data binding via the `'tabris-decorators'` extension, which is a separate module from `'tabris'`. **This extension works only with TypeScript/JSX projects**. Mixed projects work as well, but the modules using the extension need to be `.ts` or `.tsx` files. When generating a Tabris.js TypeScript project using the [Tabris CLI](https://www.npmjs.com/package/tabris-cli) `init` command an example app may be chosen that already has `'tabris-decorators'` installed. For manual installation follow the steps below.
+
+## Setup
+
+To install the `tabris-decorators` module itself type the following in your existing TypeScript-based (i.e. [tsc-compiled](../runtime.md#compiled-javascript-projects)) project:
+
+`npm install tabris-decorators`.
+
+Check your `tsconfig.json` to ensure the compiler options `experimentalDecorators`, `emitDecoratorMetadata`, `jsx` and `jsxFactory` options are set as below:
+
+```js
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    /* your other options... */
+    "jsx": "react",
+    "jsxFactory": "JSX.createElement",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+  },
+  "include": [
+    "./src/**/*.ts",
+    "./src/**/*.tsx"
+  ]
+}
+```
+
+You can now start using `tabris-decorators` in your project. It's recommended to familiarize yourself with the basic [TypeScript decorators syntax](https://www.typescriptlang.org/docs/handbook/decorators.html#class-decorators) if you aren't already.
+
```

Comment:
Maybe move this to a `doc/index.html` linking to databinding and DI docs to prevent duplication?

Reply:
Shortened the introduction and linked to the TypeScript article instead